tests/utils.ts

```diff
@@ -10,3 +11,19 @@ export async function logTx(provider: Provider, txId: string, verbose: boolean =
   }
 };
 
+
+export async function execute(
+  provider: Provider,
+  instructions: TransactionInstruction[],
+  signers: Signer[],
+  skipPreflight: boolean = false
+): Promise<String> {
+  let tx = new Transaction();
+  instructions.map((ix) => { tx = tx.add(ix) });
+  const txid = await provider.send(tx, signers, {
+    commitment: "confirmed",
+    skipPreflight,
+  });
+  await logTx(provider, txid, false);
```

Comment:
Maybe make this optional?

Reply:
actually `logTx` is needed to make sure the transaction succeeded after it was confirmed